components/suggestion/search_date_suggestion/search_date_suggestion.jsx

```diff
@@ -22,11 +43,19 @@ export default class SearchDateSuggestion extends Suggestion {
                 today: this.props.currentDate,
             };
         }
+
+        let currentLocale;
+        if (this.props.currentLocale) {
```

Comment:
Did you ever run into a case where this was empty? I think `getCurrentLocale` always returns a default value

Reply:
I do not think `currentLocale` would be empty with the current code. I added the if statement as a sanity check to make the code less coupled.  Do you suggest me to remove it?

Reply:
It's fine either way. I was mostly curious if you had run into something like that since it wouldn't be the first time something was null when I expected it to always be set.